Resources/public/css/views/userprofile.css

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-userprofileview {
+    position: absolute;
+    top: 5px;
+    right: 0;
+}
+
+.is-user-profile-hidden {
+    display: none;
+}
+
+
+.ez-view-userprofileview .ez-user-profile {
+    padding: 0.5em 2em 0 1em;
+    margin: 0;
+    position: relative;
+}
+
+.ez-view-userprofileview .ez-user-profile .ez-user-profile-info {
+    display: inline-block;
+}
+
+.ez-view-userprofileview .ez-user-profile .ez-user-profile-name {
+    margin: 0;
+}
+
+.ez-view-userprofileview .ez-user-profile-image {
+    width: 2em;
```

Comment:
since it's a bitmap image, I think its size should be set in pixels and to the chosen value in image_variations.yml.